src/ansys/mapdl/core/launcher.py

```diff
@@ -1678,14 +1717,19 @@ def launch_mapdl(
 
     # Setting number of processors
     machine_cores = psutil.cpu_count(logical=False)
-    if not nproc:
-        if machine_cores < 2:  # default required cores
-            nproc = machine_cores  # to avoid starting issues
+    if not ON_SLURM:
+        # Bypassing number of processors checks because VDI/VNC might have
+        # different number of processors than the cluster compute nodes.
+        if not nproc:
```

Comment:
`not nproc` will always evaluate to false now

Reply:
You are right.

Reply:
Actually not... the ``nproc`` is ``None`` by default. If we are on slurm (``ON_SLURM`` == ``True``), then it will have a value. Otherwise, it will be ``None`` yet.

So I'm keeping this as it was.